mix.lock

```diff
@@ -1,15 +1,19 @@
 %{
```

Comment:
This file needs to be added to .gitignore 

It contains the loaded versions of your dependencies locally, so your mix can know whether to fetch/update deps or not.